packages/portal-proto/src/features/homepage/Bodyplot.tsx

```diff
@@ -177,12 +180,13 @@ export const Bodyplot = (): JSX.Element => {
     [],
   );
 
+  const { width } = useViewportSize();
   useMemo(() => {
     if (bodyplotRef?.current) {
       createHumanBody({
         title: "Cases by Major Primary Site",
         selector: bodyplotRef.current,
-        width: 500,
+        width: width < 769 ? 400 : 500,
```

Comment:
Does this work?
`width: width >= tailwindConfig.theme.extend.screens.md ? 500 : 400`

Reply:
Just fixed this! 👍